### DISCUSSION TITLE
Reordering

### DISCUSSION THREAD

**Discussion by JacquesLG** [Okipa/laravel-table#122] [Q&A] [answered]
Hello,

It seems to be missing something in the documentation about reordering.
The documentation mentions some javascript to add but doesn't talk about it more than that.

Would you be able to help?

**Comment by Okipa** [comment, marked as answer]
Hi @JacquesLG,

You'll find all information in the package doc: https://github.com/Okipa/laravel-table#allow-columns-to-be-reordered-from-drag-and-drop-action

You have two things to setup before using reordering:
* [You'll have to set up a few lines of javascript](https://github.com/Okipa/laravel-table#set-up-a-few-lines-of-javascript) to allow reorder action feedback to be working properly
* You'll have to install the [Livewire Sortable Plugin](https://github.com/livewire/sortable), that will handle the drag and drop utility for us

Then follow the instructions specified in the doc and you should be fine!
